### DISCUSSION TITLE
IDH Honeypot - File store on Fake FTP/WEB Servers by auth. Attackers

### DISCUSSION THREAD

**Discussion by Ax386** [Security-Onion-Solutions/securityonion#8016] [Unsupported Versions] [answered]
Hi,
does anyone already set up the IDH Honeypot with file storage for authenticated sessions (attacker can login on IDH) ?

... we fully integrated the IDH in an distributed SO architecture - works very fine with playbook integration 

would be great if there are some experience how to store files for authenticated attacker with any user/pass combination

thanks a lot in advance!
Greets,
Alex

**Comment by defensivedepth** [comment, marked as answer]
What protocol specifically are you looking to mimic?

**Reply by Ax386** [reply]
a web folder ... or ftp ..i was looking for..

**Reply by defensivedepth** [reply]
The IDH node is built on OpenCanary - https://github.com/thinkst/opencanary

I would suggest checking out the features of OpenCanary - most of them are supported in the IDH node.

**Reply by Ax386** [reply]
hi,
thanks a lot for reply.
i already checked out git, and the opencanary site a well...
do you know is there some deeper documentation focus on SO integrations available? ..
i also look for real basic documentations, like changing skins on the webserver module etc..
sorry for kind of neeby questions... 
thx, Alex

**Reply by defensivedepth** [reply]
@Ax386 Have looked over our documentation?

https://docs.securityonion.net/en/2.3/idh.html#custom-configuration

**Reply by Ax386** [reply]
yes, in the meanwhile ..a couple of 100 hours i spent on SO documentation.. :-) .. we are running our SOC on SO -  
..great work the new dashboard, by the way!
thanks anyway for your help! should you find something regarding changing skins on IDH, would be great if you share it. 
 Thx, Alex

**Reply by defensivedepth** [reply]
Per the documentation (https://docs.securityonion.net/en/2.3/idh.html#custom-configuration), all service defaults can be changed. 

The http skin for example: https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/idh/defaults/http.defaults.yaml#L7
